src/MapfileStyleParser.ts

```diff
@@ -628,28 +631,75 @@ export class MapfileStyleParser implements StyleParser {
   /**
    * Get the GeoStyler-Style RasterSymbolizer from an Mapfile STYLE.
    *
-   * @param {object} styleParameters The Mapfile Style Parameters
+   * @param {object} mapfileLayer The Mapfile Style Parameters
    * @return {RasterSymbolizer} The GeoStyler-Style RasterSymbolizer
    */
-  getRasterSymbolizerFromMapfileStyle(styleParameters: MapfileStyle): RasterSymbolizer {
+  getRasterSymbolizersFromMapfileLayer(mapfileLayer: MapfileLayer): RasterSymbolizer {
     const rasterSymbolizer = { kind: 'Raster' } as RasterSymbolizer;
 
-    if (styleParameters.opacity) {
-      rasterSymbolizer.opacity = styleParameters.opacity / 100;
+    if (mapfileLayer.composite && mapfileLayer.composite.opacity) {
+      rasterSymbolizer.opacity = mapfileLayer.composite.opacity / 100;
     }
-    /*
-    if (styleParameters.resamplingMethod) {
-      const resamplingMethod = styleParameters.resamplingMethod;
-      switch (resamplingMethod.toLowerCase()) {
-        case 'average':
-          rasterSymbolizer.resampling = 'linear';
-          break;
-        case 'nearest':
-          rasterSymbolizer.resampling = 'nearest';
-          break;
+
+    if (mapfileLayer.processings) {
+      const processings: any = {};
+      mapfileLayer.processings.forEach((element) => {
+        const parts = element.split('=');
+        processings[parts[0].toLowerCase()] = parts[1].toLowerCase();
+      });
+
+      // TODO: is this mapping ok?
```

Comment:
I think so but I'm not sure what GeoStyler understands by "linear". I haven't found any doc on this... It could also mean "linear average" which would be an average of the surrounding values... In this case the "average" case could also be considered "linear"...

Reply:
Ok, removed the TODO.

Reply:
I don't know.
I propose to merge it like that, and after the first release,  grep TODO in code to make issues and prioritize the tasks.